public-webinars/vscode-extension/lesson-plan.md

```diff
@@ -0,0 +1,65 @@
+# VSCode webinar
+
+**Goals:** Viewers will leave the day with an understanding of how the Sourcegraph for VSCode extension works, they will have installed in themselves, and they will have run at least one search.
+
+## Intro
+
+Today I'm going to be walking through our VSCode for Sourcegraph extension, and highting how you can configure it to search open source code—or your own private code—right from VSCode, without needing to have that code pulled down locally. By the end of today, I want to make sure everyone leaves here with an understanding of why we built the extension, how to install it, and how to use it—and hopefully, everyone will leave here having installed it themselves!
+
+## History
+
+For those of you who may not be familiar with Sourcegraph, first of all: welcome! Secondly, a brief overview of what we are. We're a code search and intelligence platform, allowing you to search across all of the code that's indexed on a particular Sourcegraph instance, in the cloud. In practice, that means you can use sourcegraph.com/search to search any OSS code you want, no matter what you have local to your machine—and if you set up a Sourcegraph instance yourself, you can search all of the code you have access to in your instance without needing to have it locally.
```

Comment:
> , no matter what you have local to your machine—and if you set up a Sourcegraph instance yourself, you can search all of the code you have access to in your instance without needing to have it locally.

Since we're limiting API access to enterprise only, we no longer can connect to your self hosted local Sourcegraph instance as well. You have to upgrade in order to use VSCode IDE on your private code.